src/Integration.Vsix/SonarLintDaemon/SonarLintDaemonSplashscreen.xaml

```diff
@@ -1,16 +1,16 @@
 ﻿<Window x:Class="SonarLint.VisualStudio.Integration.Vsix.SonarLintDaemonSplashscreen"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        Title="Activate support for JavaScript in SonarLint" Width="500"
+        Title="Activate support for additional languages in SonarLint" Width="500"
         WindowStartupLocation="CenterScreen" Topmost="True" SizeToContent="Height"
         >
     <StackPanel Margin="5">
         <TextBlock Margin="0,5"
-            Text="SonarLint for Visual Studio can also analyze JavaScript files."
+            Text="SonarLint for Visual Studio can also analyze JavaScript, C or C++ files."
```

Comment:
@henryju I think that we should replace `or` by `and` here for consistency with other places. Also maybe in all places would make sense to swap `C` and `C++`, so that `C++` comes first as more frequently used and hence more expected by users integration. Maybe even swap with `JavaScript`.